test/smoke/smoke_test.go

```diff
@@ -56,7 +56,7 @@ var _ = Describe("Smoke test", func() {
 			testenv.StandaloneReady(deployment, deployment.GetName(), standalone, testenvInstance)
 
 			// Verify MC Pod is Ready
-			testenv.MCPodReady(testenvInstance.GetName(), deployment)
+			// testenv.MCPodReady(testenvInstance.GetName(), deployment)
```

Comment:
As clarified by @pdhanoyasplunk , we need to remove MC checks in the following as they are now redundant (as they are covered as part of MC testing):
* smartstore
* smoke
* scaling
To be taken up as part of another task

Reply:
Task Created CSPL-1400